modules/testing/index.js

```diff
@@ -25,12 +25,12 @@ export const makeProject = async (config, callback) => {
       }),
       vite: {
         server: {
-          watch: {
-            // During tests we edit the files too fast and sometimes chokidar
-            // misses change events, so enforce polling for consistency
-            usePolling: true,
-            interval: 100,
-          },
+          // watch: {
```

Comment:
1) Dont commit commented out code
2) This is completely out of scope of this PR. If it helped, create new PR so it can be tested separately.